app/data/tasks.json

```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "id": 1,
+    "user_id": 1,
+    "name": "Livin' easy",
+    "date": "today",
+    "status": "active"

```

Comment:
Please, add fields for task like
'from', 'to' - date
'duration'  - time (hours)
also 
'number'

Reply:
'witch be show' - please, enter it into Google Translate :)